requirements/edx-sandbox/py35.in

```diff
@@ -7,8 +7,6 @@
 #   * confirm that it has no system requirements beyond what we already install
 #   * run "make upgrade" to update the detailed requirements files
 
--c ../constraints.txt
```

Comment:
Is this risky to have it out of sync?  Not sure how this is used.

Reply:
We're moving away from keeping codejail dependencies synchronized with the LMS/Studio ones, since we can't upgrade the former very often without risking breakage of grader code.